app/services/runner_trace.rb

```diff
@@ -1,4 +1,5 @@
 class RunnerTrace
```

Comment:
Let's remove this `Runner` now?

Reply:
Yup, I will in a follow up PR 👍 ....just want to make sure everything is working for a couple days before removing the tracer.